<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PatientRegistration</class>
 <widget class="QWidget" name="PatientRegistration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1117</width>
    <height>1037</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
    font-family: century gothic;
    font-size: 14px;
}

QToolButton
{
    background: #D1D2D2;
    border-radius: 30px;
}

QLabel
{
    color: black;
}

QPushButton
{
   border: 2px solid black;
   background-color: #D1D2D2;
   border-radius: 15px;
}

QPushButton:hover
{
   color: #D1D2D2;
   border-radius: 15px;
   background: #16A5C3;
}

QFrame
{
  background: #C7D1D1;
  border: 2px solid black
}
QWidget
{
   background-color:   #6dd5ed;
}

QMessageBox
{
   color: black;
   background-color: white;
}</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>740</y>
     <width>351</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame
{
   background: #9db0ad;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>5</y>
      <width>329</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>New Registration</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="patient_add_record_btn">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>840</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
    color: black;
    background-color: white;
}

QPushButton:hover
{
   color: #D1D2D2;
   border-radius: 15px;
   background: #426a78;
}</string>
   </property>
   <property name="text">
    <string>Add Record</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>0</y>
     <width>311</width>
     <height>311</height>
    </rect>
   </property>
   <property name="title">
    <string>Patient Personal Information</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="FNameLabel">
       <property name="styleSheet">
        <string notr="true">QLabel
{
    color: black;
    background-color: white;
}</string>
       </property>
       <property name="text">
        <string>First Name:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="FNameEdit">
       <property name="styleSheet">
        <string notr="true">QLineEdit
{
    color: black;
    background-color: white;
}</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="LNameLabel">
       <property name="styleSheet">
        <string notr="true">QLabel
{
    color: black;
    background-color: white;
}</string>
       </property>
       <property name="text">
        <string>Last Name:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="LNameEdit">
       <property name="styleSheet">
        <string notr="true">QLineEdit
{
    color: black;
    background-color: white;
}</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">QLabel
{
    color: black;
    background-color: white;
}</string>
       </property>
       <property name="text">
        <string>Telephone #:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="styleSheet">
        <string notr="true">QLineEdit
{
    color: black;
    background-color: white;
}</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter_23">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="Email">
       <property name="styleSheet">
        <string notr="true">QLabel
{
    color: black;
    background-color: white;
}</string>
       </property>
       <property name="text">
        <string>Email</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="DOBEdit_2">
       <property name="styleSheet">
        <string notr="true">QLineEdit
{
    color: black;
    background-color: white;
}</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="DOBLabel">
       <property name="styleSheet">
        <string notr="true">QLabel
{
    color: black;
    background-color: white;
}</string>
       </property>
       <property name="text">
        <string>Date of Birth:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="DOBEdit">
       <property name="styleSheet">
        <string notr="true">QLineEdit
{
    color: black;
    background-color: white;
}</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter_6">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="GenderLabel">
       <property name="styleSheet">
        <string notr="true">QLabel
{
    color: black;
    background-color: white;
}</string>
       </property>
       <property name="text">
        <string>Gender:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="GenderCombo">
       <property name="styleSheet">
        <string notr="true">QComboBox
{
    color: black;
    background-color: white;
}</string>
       </property>
       <item>
        <property name="text">
         <string>Female</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Male</string>
        </property>
       </item>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter_7">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="PatientLabel">
       <property name="styleSheet">
        <string notr="true">QLabel
{
    color: black;
    background-color: white;
}</string>
       </property>
       <property name="text">
        <string>Patient Status:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="PatientCombo">
       <property name="styleSheet">
        <string notr="true">QComboBox
{
    color: black;
    background-color: white;
}</string>
       </property>
       <item>
        <property name="text">
         <string>In need of Blood</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>On Treatment</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Deceased</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>New Patient</string>
        </property>
       </item>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>0</y>
     <width>280</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>Patient Residences</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QSplitter" name="splitter_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="DistrictLabel">
       <property name="styleSheet">
        <string notr="true">QLabel
{
    color: black;
    background-color: white;
}</string>
       </property>
       <property name="text">
        <string>District:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="DistrictCombo">
       <property name="styleSheet">
        <string notr="true">QComboBox
{
    color: black;
    background-color: white;
}</string>
       </property>
       <item>
        <property name="text">
         <string>Orange Walk</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cayo</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Belize</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Stann Creek</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Toledo</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Corozal</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Other</string>
        </property>
       </item>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter_24">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QLabel" name="label_16">
       <property name="styleSheet">
        <string notr="true">QLabel
{
    color: black;
    background-color: white;
}</string>
       </property>
       <property name="text">
        <string>Address</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit_3">
       <property name="styleSheet">
        <string notr="true">QTextEdit
{
    color: black;
    background-color: white;
}</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>10</y>
     <width>268</width>
     <height>205</height>
    </rect>
   </property>
   <property name="title">
    <string>Blood Information</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="label_15">
      <property name="styleSheet">
       <string notr="true">QLabel
{
    color: black;
    background-color: white;
}</string>
      </property>
      <property name="text">
       <string>Blood Type</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="bloodtype_combobox"/>
    </item>
    <item>
     <widget class="QSplitter" name="splitter_12">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">QLabel
{
    color: black;
    background-color: white;
}</string>
       </property>
       <property name="text">
        <string>Blood Pressure</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="styleSheet">
        <string notr="true">QLineEdit
{
    color: black;
    background-color: white;
}</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter_14">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_5">
       <property name="styleSheet">
        <string notr="true">QLabel
{
    color: black;
    background-color: white;
}</string>
       </property>
       <property name="text">
        <string>Pulse:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_4">
       <property name="styleSheet">
        <string notr="true">QLineEdit
{
    color: black;
    background-color: white;
}</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter_13">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">QLabel
{
    color: black;
    background-color: white;
}</string>
       </property>
       <property name="text">
        <string>Heart Rate:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="styleSheet">
        <string notr="true">QLineEdit
{
    color: black;
    background-color: white;
}</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter_15">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_6">
       <property name="styleSheet">
        <string notr="true">QLabel
{
    color: black;
    background-color: white;
}</string>
       </property>
       <property name="text">
        <string>Blood Sugar</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_5">
       <property name="styleSheet">
        <string notr="true">QLineEdit
{
    color: black;
    background-color: white;
}</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>350</y>
     <width>335</width>
     <height>129</height>
    </rect>
   </property>
   <property name="title">
    <string>Risk Factors</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="title">
       <string>HyperTension</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QCheckBox" name="checkBox">
         <property name="styleSheet">
          <string notr="true">QCheckBox
{
    color: black;
    background-color: white;
    border: 2px solid black
}</string>
         </property>
         <property name="text">
          <string>Hypertension</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSplitter" name="splitter_10">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QLabel" name="hyperLabel">
          <property name="styleSheet">
           <string notr="true">QLabel
{
    color: black;
    background-color: white;
}</string>
          </property>
          <property name="text">
           <string>Years w/ Hypertension</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="hyperEdit">
          <property name="styleSheet">
           <string notr="true">QLineEdit
{
    color: black;
    background-color: white;
}</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_7">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>220</y>
     <width>421</width>
     <height>521</height>
    </rect>
   </property>
   <property name="title">
    <string>Diabetes</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <widget class="QSplitter" name="splitter_9">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QCheckBox" name="Type1Box">
       <property name="styleSheet">
        <string notr="true">QCheckBox
{
    color: black;
    background-color: white;
}</string>
       </property>
       <property name="text">
        <string>Diabetes Type 1</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="Type2Box">
       <property name="styleSheet">
        <string notr="true">QCheckBox
{
    color: black;
    background-color: white;
}</string>
       </property>
       <property name="text">
        <string>Diabetes Type 2</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter_8">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="diabetesLabel">
       <property name="styleSheet">
        <string notr="true">QLabel
{
    color: black;
    background-color: white;
}</string>
       </property>
       <property name="text">
        <string>Years w/ Diabetes</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="diabetesEdit">
       <property name="styleSheet">
        <string notr="true">QLineEdit
{
    color: black;
    background-color: white;
}</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="eyeDamageBox">
      <property name="whatsThis">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;diabetic macular edema&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">QCheckBox
{
    color: black;
    background-color: white;
    border: 2px solid black
}</string>
      </property>
      <property name="text">
       <string>Eye Damage Due to DM</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter_18">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_9">
       <property name="styleSheet">
        <string notr="true">QLabel
{
    color: black;
    background-color: white;
}</string>
       </property>
       <property name="text">
        <string>Urine-SG(Specific Gravity):</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_8">
       <property name="styleSheet">
        <string notr="true">QLineEdit
{
    color: black;
    background-color: white;
}</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter_20">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_11">
       <property name="styleSheet">
        <string notr="true">QLabel
{
    color: black;
    background-color: white;
}</string>
       </property>
       <property name="text">
        <string>Urine-Bilirubin</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_10">
       <property name="styleSheet">
        <string notr="true">QLineEdit
{
    color: black;
    background-color: white;
}</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter_11">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QCheckBox" name="checkBox_2">
       <property name="styleSheet">
        <string notr="true">QCheckBox
{
    color: black;
    background-color: white;
}</string>
       </property>
       <property name="text">
        <string>Problem Urinating?</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">QLabel
{
    color: black;
    background-color: white;
}</string>
       </property>
       <property name="text">
        <string>If yes, describe urine</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit">
       <property name="styleSheet">
        <string notr="true">QTextEdit
{
    color: black;
    background-color: white;
}</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter_21">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_12">
       <property name="styleSheet">
        <string notr="true">QLabel
{
    color: black;
    background-color: white;
}</string>
       </property>
       <property name="text">
        <string>Urine-PH:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_11">
       <property name="styleSheet">
        <string notr="true">QLineEdit
{
    color: black;
    background-color: white;
}</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter_22">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_13">
       <property name="styleSheet">
        <string notr="true">QLabel
{
    color: black;
    background-color: white;
}</string>
       </property>
       <property name="text">
        <string>Urine-Keytones:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_12">
       <property name="styleSheet">
        <string notr="true">QLineEdit
{
    color: black;
    background-color: white;
}</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter_16">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_7">
       <property name="styleSheet">
        <string notr="true">QLabel
{
    color: black;
    background-color: white;
}</string>
       </property>
       <property name="text">
        <string>Urine-Leukocytes:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_6">
       <property name="styleSheet">
        <string notr="true">QLineEdit
{
    color: black;
    background-color: white;
}</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_3">
      <property name="styleSheet">
       <string notr="true">QCheckBox
{
    color: black;
    background-color: white;
    border: 2px solid black
}</string>
      </property>
      <property name="text">
       <string>Urine-Blood</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter_19">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_10">
       <property name="styleSheet">
        <string notr="true">QLabel
{
    color: black;
    background-color: white;
}</string>
       </property>
       <property name="text">
        <string>Urine-Glucose:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_9">
       <property name="styleSheet">
        <string notr="true">QLineEdit
{
    color: black;
    background-color: white;
}</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter_17">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_8">
       <property name="styleSheet">
        <string notr="true">QLabel
{
    color: black;
    background-color: white;
}</string>
       </property>
       <property name="text">
        <string>Urine-Nitrite:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_7">
       <property name="styleSheet">
        <string notr="true">QLineEdit
{
    color: black;
    background-color: white;
}</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
